public/app/features/transformers/editors/ReduceTransformerEditor.tsx

```diff
@@ -56,61 +56,48 @@ export const ReduceTransformerEditor = ({ options, onChange }: TransformerUIProp
 
   return (
     <>
-      <div>
-        <div className="gf-form gf-form--grow">
-          <div className="gf-form-label width-8" aria-label={selectors.components.Transforms.Reduce.modeLabel}>
-            Mode
-          </div>
-          <Select
-            options={modes}
-            value={modes.find((v) => v.value === options.mode) || modes[0]}
-            onChange={onSelectMode}
-            className="flex-grow-1"
-          />
-        </div>
-      </div>
-      <div className="gf-form-inline">
-        <div className="gf-form gf-form--grow">
-          <div className="gf-form-label width-8" aria-label={selectors.components.Transforms.Reduce.calculationsLabel}>
-            Calculations
-          </div>
-          <StatsPicker
-            className="flex-grow-1"
-            placeholder="Choose Stat"
-            allowMultiple
-            stats={options.reducers || []}
-            onChange={(stats) => {
-              onChange({
-                ...options,
-                reducers: stats as ReducerID[],
-              });
-            }}
-          />
-        </div>
-      </div>
+      <InlineField label="Mode" aria-label={selectors.components.Transforms.Reduce.modeLabel} grow labelWidth={16}>
+        <Select
+          options={modes}
+          value={modes.find((v) => v.value === options.mode) || modes[0]}
+          onChange={onSelectMode}
+        />
+      </InlineField>
+      <InlineField
+        label="Calculations"
+        aria-label={selectors.components.Transforms.Reduce.calculationsLabel}
+        grow
+        labelWidth={16}
+      >
+        <StatsPicker
+          placeholder="Choose Stat"
+          allowMultiple
+          stats={options.reducers || []}
+          onChange={(stats) => {
+            onChange({
+              ...options,
+              reducers: stats as ReducerID[],
+            });
+          }}
+        />
+      </InlineField>
       {options.mode === ReduceTransformerMode.ReduceFields && (
-        <div className="gf-form-inline">
-          <div className="gf-form">
-            <LegacyForms.Switch
-              label="Include time"
-              labelClass="width-8"
-              checked={!!options.includeTimeField}
-              onChange={onToggleTime}
-            />
-          </div>
-        </div>
+        <>
+          <LegacyForms.Switch
```

Comment:
Great call because there IS a replacement 🎉 
Replaced it with `InlineSwitch` now. Top is InlineSwitch, bottom is legacy Switch:
<img width="218" alt="Screenshot 2023-07-10 at 12 04 39" src="https://github.com/grafana/grafana/assets/100691367/d87e5f1b-0119-4163-991c-befb4b83e792">